components/iam/pkg/oidc/router_test.go

```diff
@@ -92,7 +92,8 @@ type testServerParams struct {
 
 func newTestServer(t *testing.T, params testServerParams) (url string, state *StateParam) {
 	router := chi.NewRouter()
-	oidcService := NewService()
+	sessionServerAddress := newFakeSessionServer(t)
```

Comment:
In line 78 above: We should be able to see the session cookie there, right? Would it make sense to check for it's presense, then?

Reply:
I guess you are right. Though it seems to be a test of the mock, it sounds like we can verify that the mock is actually called and Set-Cookie is carried on properly.

Reply:
done